### ISSUE TITLE
Lumen 5.5 - ReflectionException: Class router does not exist in Folklore/GraphQL/ServiceProvider.php

### ISSUE THREAD

**Issue by spectrumcat** [Folkloreatelier/laravel-graphql#200]
Hi,

The exact error I get is:

```
ReflectionException
Class router does not exist
--
in Container.php (line 752)
at ReflectionClass->__construct('router')in Container.php (line 752)
at Container->build('router')in Container.php (line 631)
at Container->resolve('router', array())in Container.php (line 586)
at Container->make('router', array())in Application.php (line 230)
at Application->make('router')in Container.php (line 1195)
at Container->offsetGet('router')in ServiceProvider.php (line 54)
at ServiceProvider->Folklore\GraphQL\{closure}(object(SchemaAdded))in Dispatcher.php (line 349)
at Dispatcher->Illuminate\Events\{closure}('Folklore\\GraphQL\\Events\\SchemaAdded', array(object(SchemaAdded)))in Dispatcher.php (line 200)
```

Using Xdebug i managed to narrow this error down to the
file: vendor/folklore/graphql/src/Folklore/GraphQL/ServiceProvider.php
line: 54

If I replace `$this->app['router']` with `$this->app->router->` I will get the following error:
```
FatalThrowableError
Call to undefined method Laravel\Lumen\Routing\Router::pattern()
```

Please help

**Comment by spectrumcat** [comment]
In older version (19) it looks like the LumenServiceProvider was overriding the `boot()` function and as a result the `bootEvents()`, that causes the error, was never called (in case of Lumen). In the current version the whole `boot()` function is not overridden any more, causing the `bootEvents()` to fire.

**Comment by dmongeau** [comment]
Should be fixed in 1.0.27

**Comment by spectrumcat** [comment]
Works great, thank you =)
